Gemfile.lock

```diff
@@ -243,7 +245,7 @@ GEM
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    active_hash (3.0.0)
+    active_hash (3.2.1)
```

Comment:
### [active_hash](http://github.com/active-hash/active_hash)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.0.0 → 3.2.1

[_[change-log](https://github.com/active-hash/active_hash/blob/master/CHANGELOG.md), [source-code](http://github.com/active-hash/active_hash), [gem-diff](https://my.diffend.io/gems/active_hash/3.0.0/3.2.1)_]





<details>
<summary>Commits</summary>

A change of **171** commits. See the full changes on [the compare page](https://github.com/active-hash/active_hash/compare/v3.0.0...v3.2.1).

These are the first 10 commits:
- (eb4d668) [Add spec for .order method](https://github.com/active-hash/active_hash/commit/eb4d668c4e171ebea4b9088b582a5f5bd96fb2a4)
- (609ffb8) [Add .order method](https://github.com/active-hash/active_hash/commit/609ffb82fb25f52e86db506937b665080f1b0144)
- (338d262) [Update README](https://github.com/active-hash/active_hash/commit/338d262df7dd5c9d764d535694e0a265a28ed7a9)
- (b19a8de) [Merge branch 'master' into add_order_method](https://github.com/active-hash/active_hash/commit/b19a8de19decfd5352908ddb212a9ef2dca51295)
- (fcfa0ab) [Modify .order method to support .where chain.](https://github.com/active-hash/active_hash/commit/fcfa0ab6edbbead9855f4606c3a0b8c4d99431a2)
- (b3edbba) [Add #to_ary to ActiveHash::Relation](https://github.com/active-hash/active_hash/commit/b3edbba09d65daedbef111f2f527591262e612ab)
- (523846a) [Use anonymous model class in spec](https://github.com/active-hash/active_hash/commit/523846ab4bf3b02c75bcf5e3fc74ecbdc105d6f6)
- (db5d0ad) [Remove #to_a calls in base_spec.rb for Relation comparisons](https://github.com/active-hash/active_hash/commit/db5d0adb6c69be6d55a2707fea5f3fbe87b40dca)
- (46a7985) [Make #find behave like its Enumerable equivalent if id is ni…](https://github.com/active-hash/active_hash/commit/46a79852f07d9fc3ab601dc43846aa28883c300c)
- (ce1da1b) [Update README.md](https://github.com/active-hash/active_hash/commit/ce1da1bbc53daa134ff20b2c30f73e2c3a6c2054)


</details>